lib/elixir/lib/io/ansi.ex

```diff
@@ -21,6 +21,20 @@ defmodule IO.ANSI do
   [ANSI escape sequences](https://en.wikipedia.org/wiki/ANSI_escape_code)
   are characters embedded in text used to control formatting, color, and
   other output options on video text terminals.
+
+  ## Examples
+
+  Because the ANSI escape sequences are embedded in text, the normal usage of
+  these functions is to concatenate their output with text.
+
+      formatted_text = IO.ANSI.blue_background() <> "Example" <> IO.ANSI.reset()
+      IO.puts formatted_text
```

Comment:
Nitpick: format code as the formatter would.

```suggestion
      IO.puts(formatted_text)
```

Reply:
+1